app/_src/quickstart/universal-demo.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Deploy Kuma on Universal
+---
+
+To start learning how {{site.mesh_product_name}} works, you run and secure a simple demo application that consists of two services:
+
+- `demo-app`: a web application that lets you increment a numeric counter. It listens on port 5000
+- `redis`: data store for the counter
+
+
+{% mermaid %}
+---
+title: service graph of the demo app
+---
+flowchart LR
+demo-app(demo-app :5000)
+redis(redis :6379)
+demo-app --> redis
+{% endmermaid %}
+
+## Prerequisites
+
+* [Redis installed](https://redis.io/docs/getting-started/)
+* Optional: To explore traffic metrics with the demo app, you also need to [set up Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/). See the [traffic metrics policy documentation](/docs/{{ page.version }}/policies/traffic-metrics).
+
+## Install {{site.mesh_product_name}}
+
+{% tabs install useUrlFragment=false %}
+{% tab install Docker %}
+Install a {{site.mesh_product_name}} control plane using the Docker images:
+
+* **kuma-cp**: at `docker.io/{{ site. mesh_docker_org }}/kuma-cp:{{ page.latest_version }}`
+* **kuma-dp**: at `docker.io/{{ site. mesh_docker_org }}/kuma-dp:{{ page.latest_version }}`
+* **kumactl**: at `docker.io/{{ site. mesh_docker_org }}/kumactl:{{ page.latest_version }}`
+
+You can freely `docker pull` these images to start using {{site.mesh_product_name}}, as we will demonstrate in the following steps.
+{% endtab %}
+{% tab install Operating Systems %}
+Do one of the following to download {{site.mesh_product_name}}:
+
+* Run the following script to automatically detect the operating system (Amazon Linux, CentOS, RedHat, Debian, Ubuntu, and macOS) and download {{site.mesh_product_name}}:
+    <div class="language-sh">
+    <pre class="no-line-numbers"><code>curl -L {{site.mesh_download_url}} | VERSION={{ page.latest_version }} sh -</code></pre>
+    </div>
+* <a href="https://packages.konghq.com/public/{{site.mesh_product_name_path}}-binaries-release/raw/names/{{site.mesh_product_name_path}}-{{ page.os }}-{{ page.arch }}/versions/{{ page.latest_version }}/{{site.mesh_product_name_path}}-{{ page.latest_version }}-{{ page.os }}-{{ page.arch }}.tar.gz">Download</a> the distribution manually. Then, extract the archive with: `tar xvzf {{site.mesh_product_name_path}}-{{ page.latest_version }}-{{ page.os }}-{{ page.arch }}.tar.gz`.
+{% endtab %}
+{% endtabs %}
```

Comment:
maybe we could just point to https://deploy-preview-1682--kuma.netlify.app/docs/2.6.x/production/install-kumactl/#install-kumactl as a prerequisite

Reply:
@slonka Since this is a quickstart page, users like to have all the info in one page so its faster/easier to get started, so I'd like to keep the install instructions.